src/main/resources/application-cloud.yml

```diff
@@ -47,6 +62,11 @@ server:
       - TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
       - TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
       - TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384
+  error:
```

Comment:
Die Settings sind in keinem anderen Projekt gesetzt. Brauchen wir die?

Reply:
I would keep the settings